### DISCUSSION TITLE
Sharable forms for adding content to collections

### DISCUSSION THREAD

**Discussion by ihadabs** [directus/directus#18807] [Feature Requests]
### Summary

The ability to create a form link for a collection to accept submissions.

### Ex 1
A team built an app for popular local restaurants. They manage the app content thru Directus. They want to allow people to submit restaurants to be reviewed and published on the app.

### Ex 2
A team built an app and they want to manage their app feedback in Directus.

### Ex 3
A team built an app for local clubs, clubs need to submit a form for the team.

### Ex 4
A team manages their landing page content thru Directus, they want to create a form for people to submit testimonials.

### Detailed Design

_The feature could be implemented this way:_

### New to DB: directus_forms
1. directus_forms
    a. Which will be used to create forms, each form is related to a collection
    b. A form UI is similar to the collection item UI
2. directus_form_links
    a. Each form can have multiple links
    b. A link could be enabled/disabled
    c. A link could have an expiry date

### New to UI: Tab for forms
1. A tab to manage forms
2. It could be laid out the same as the `Content` tab
3. When you click on a collection you will be able to see its forms
4. When you click on a form you will see its fields, and links
5. Form submissions are regular items in the collection (we can relate items add by a form, this need more db tables) 

### Must Have:
- Collection Form(s)
- Collection Form(s) Link(s)

### Drawbacks

The feature itself could start very small.

But it could be very complicated if we wanted to cover more use cases.

### Alternatives

1. Using Google Forms, or Typeform.
2. Each team has to create their UI for adding things to a collection.

### Adoption Strategy

1. As this is a completely new thing, nobody will need to change a thing.
2. It could be marketed as:
    a. Direcuts users can now accept editors to their collection without creating users for them.
    b. Replace Typeform, Google Forms, and Airtable for some use cases.

**Comment by benhaynes** [comment]
Yes, please!! ⏫

**Comment by lpil** [comment]
This would be super lovely to have!

Does anyone have a third-party approach to this in the mean time?

**Comment by maazamaani** [comment]
I think it could be merged with "share item" feature. We can share editable or viewable links to others.
